test/validators/movieCreate.test.js

```diff
@@ -2,31 +2,31 @@
 
 const Joi = require('joi');
 
-const MovieValidator = require('../../lib/validators/movie');
+const MovieCreateValidator = require('../../lib/validators/movieCreate');
 
 describe('movie validator', () => {
```

Comment:
To be more descriptive in the test output, you should change this to `movie create validator`.